webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -647,7 +648,90 @@ private static void addFmcSolutionSheet(PdfWriter docWriter, Document doc, Scram
 
             offsetTop += fontSize + marginBottom;
         }
+        
+        // the moves specs inside the table are not exactly with the rules.
+        // they are rendered separately, then rules are added above it
+        
+        // table
```

Comment:
nit: The formatting of this comment feels a bit off... could you clean this up and write in full sentences (with capitalization and punctuation).

More importantly, I don't understand what this comment is telling me, or which code it's actually referring to. What's a "move spec"?

Reply:
I removed this part, it was a bit confusing. "Move spec" is the table itself. Since the table is not a text, it has to be rendered then placed.